gotechnica/platform-2020
# Made text changes to the platform

# Description 
1. On the team formation page, added a sentence: "You still need to create a team, even if you are a hacker working alone."
1. Replace “organizer” with “representative” on home page link section

# Steps to test
1. cd into the frontend
1. run `npm run serve`
1. make sure organizer is replaced with representative
1. go to "Teams" page, make sure there is a line that says: "You still need to create a team, even if you are a hacker working alone."
